File: src/main/kotlin/b_basic_conditions/exercise_20.kt
package b_basic_conditions

// Develop a program that reads an integer and shows whether it is EVEN or ODD.
fun evenOrOdd() {
    print("Number: ")
    val number = readln().toInt()

    if (number % 2 == 0) {
        println("It's even!")
    } else {
        println("It's odd!")
    }
}

File: Dart/b_basic_conditions/exercise_20.dart
// Develop a program that reads an integer and shows whether it is EVEN or ODD.

import 'dart:io';

void main() {
  print('ODD OR EVEN');
  print('----------------------------');

  stdout.write('Number: ');
  int number = int.parse(stdin.readLineSync()!);

  if (number % 2 == 0) {
    print('it\'s even!');
  } else {
    print('it\'s odd!');
  }
}
